### ISSUE TITLE
Propose a small spelling correction to Laziness ?

### ISSUE THREAD

**Issue by jeis2497052** [PyGithub/PyGithub#686]
Hi All,
did you want to review Laziness ?

```
diff --git a/doc/Design.md b/doc/Design.md
index 7863d93a..1971f3b6 100644
--- a/doc/Design.md
+++ b/doc/Design.md
@@ -22,7 +22,7 @@ They
    - may be GET-able (to ask if an object is in the list)
    - may be DELETE-able (to remove an object from the list)
 
-Lazyness
+Laziness
 ========
 

```

**Comment by jasonwhite** [comment]
A PR would be much appreciated!

**Comment by sfdye** [comment]
Closed in #701
